__tests__/api/AutoBundler.test.ts

```diff
@@ -171,4 +171,19 @@ describe("AutoBundler01", () => {
         // Check the output as JSON
         expect(JSON.stringify(ab.getBundle().getManifest())).toMatch("{\"manifest\":{\"xmlns\":\"http://www.ibm.com/xmlns/prod/cics/bundle\",\"bundleVersion\":1,\"bundleRelease\":0}}");
     });
+    it("should cope with an empty directory and generate a NODEJSAPP", () => {
+
+        let parms: IHandlerParameters;
+        parms = DEFAULT_PARAMTERS;
+        parms.arguments.port = 500;
+        parms.arguments.nodejsapp = "wibble";
+        parms.arguments.startscript = "__tests__/__resources__/EmptyBundle02/META-INF";
```

Comment:
Why set startscript to point to a META-INF dir?

Reply:
I needed something to point it at otherwise the test would fail with a missing startscript error. Perhaps I was being lazy in using META-INF for that purpose, it's just an arbitrary value.